packages/Popover/src/components/Content/Content.js

```diff
@@ -9,14 +11,16 @@ import { ContentStyled } from "./Content.styles";
 
 const propTypes = {
   children: PropTypes.node,
+  isAlwaysOnDom: PropTypes.bool,
 };
 
 const defaultProps = {
   children: null,
+  isAlwaysOnDom: false,
```

Comment:
Added this prop for backward compatibility, so if there is a situation where it is much better to keep some popover content on the dom (perhaps if there is content that needs to render complex components, or make fetches to the server), they would need to add this prop

Reply:
I think we should align with the pre-existing `shouldUnmount` prop name we have on the `<Tabs.Panel>` component:

https://github.com/acl-services/paprika/blob/master/packages/Tabs/src/components/Panel/Panel.js#L13

(Even though it means the default value should be `true`)

Reply:
I think this prop should apply to both `Tip` and `Content`, not just on the `Content` component. Otherwise if `isAlwaysOnDom` is `true` on `Content`, the `Tip` will still unmount which seems a little bit strange to me. But if the team agree with it, I'm totally fine 😄